src/stringify.js

```diff
@@ -124,7 +123,35 @@ var BASE_16 = 16;
         var mind = gap;
         var partial;
         var value = holder[key];
-        var isInt64 = value !== null && value instanceof Int64;
+        var isInt64 = true;
+        var Int64Keys = ["buffer", "offset"];
+        var Int64Types = [Uint8Array, Number];
+        var valueKeys = [];
```

Comment:
this seems to be only used inside the branch, can't we move the declaration there?